.github/workflows/buf.yaml

```diff
@@ -23,3 +23,14 @@ jobs:
           buf_token: ${{ secrets.BUF_TOKEN }}
           create_visibility: public
           draft: ${{ github.ref_name != 'main'}}
+  lint-module:
```

Comment:
instead of using `branches-module` as the example here we can follow the configuration for `breaking-module`, so we'll end up with:
```
  lint-module:
    runs-on: ubuntu-latest
    if: github.ref == 'refs/heads/main'
    steps:
      - uses: actions/checkout@v3
      - uses: bufbuild/buf-setup-action@v1
      - uses: bufbuild/buf-push-action@v1
        with:
          input: finish/tutorial-lint/proto
          buf_token: ${{ secrets.BUF_TOKEN }}
```

the key differences between this^ and what is in the PR:
- we're only pushing to the BSR when we've made a proto change to this module on the main git branch. we wanted drafts created on the branches module for the purposes of that tutorial (this is indicated by `draft: ${{ github.ref_name != 'main'}})`. we don't care about drafts/branches in the context of the lint tutorial so we only care about changes on main, so we've removed the `draft` clause and added `if: github.ref == 'refs/heads/main'`.
- we want the `input` to be finish/tutorial-lint/proto instead of start/tutorial-lint/proto since that's where the proto changes we care about in this PR are